### ISSUE TITLE
cmd/compile: internal compiler error: method ReleaseMessage on *uint8 not found

### ISSUE THREAD

**Issue by k-pa** [golang/go#57453]
### What version of Go are you using (`go version`)?

<pre>
$ go version go version go1.19 windows/amd64
</pre>

### Does this issue reproduce with the latest release?
Yes

### What operating system and processor architecture are you using (`go env`)?

Windows 11 on a 64bit intel i7 processor

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
set GO111MODULE=
set GOARCH=amd64
set GOBIN=
set GOCACHE=C:\Users\myself\AppData\Local\go-build
set GOENV=C:\Users\myself\AppData\Roaming\go\env
set GOEXE=.exe
set GOEXPERIMENT=
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOINSECURE=
set GOMODCACHE=C:\Users\myself\go\pkg\mod
set GOOS=windows
set GOPATH=C:\Users\myself\go
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=C:\Program Files\Go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=C:\Program Files\Go\pkg\tool\windows_amd64
set GOVCS=
set GOVERSION=go1.19
set GCCGO=gccgo
set GOAMD64=v1
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=C:\Data\tests\coap-server\go.mod
set GOWORK=
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fno-caret-diagnostics -Qunused-arguments -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=C:\Users\myself\AppData\Local\Temp\go-build584022713=/tmp/go-build -gno-record-gcc-switches

</pre></details>

### What did you do?

I am trying to build [this coap server example](https://github.com/plgd-dev/go-coap/blob/master/examples/dtls/psk/server/main.go) under Windows with go1.19.3 but get the following error:
`internal compiler error: method ReleaseMessage on *uint8 not found
`
I also tried with go 1.18.9 on windows with same result

I tried the same code under Linux with go 1.18.9 and it works perfectly 

### What did you expect to see?

No build issue as under Linux

### What did you see instead?

```
go build
# github.com/plgd-dev/go-coap/v3/tcp/client
C:\Users\myself\go\pkg\mod\github.com\plgd-dev\go-coap\v3@v3.0.1\net\observation\handler.go:130:57: internal compiler error: method ReleaseMessage on *uint8 not found

Please file a bug report including a short program that triggers the error.
https://go.dev/issue/new
# github.com/plgd-dev/go-coap/v3/udp/client
C:\Users\myself\go\pkg\mod\github.com\plgd-dev\go-coap\v3@v3.0.1\net\observation\handler.go:130:57: internal compiler error: method ReleaseMessage on *uint8 not found

Please file a bug report including a short program that triggers the error.
https://go.dev/issue/new
```

**Comment by ALTree** [comment]
This is #55270 and it should has been fixed in 1.19.2. 

Are you sure you are reproducing this on 1.19.3 ? `go version` and your `go env` output show you're using 1.19.0, not .3

**Comment by k-pa** [comment]
@ALTree I just installed 1.19.4 manually and it works

Sorry for the noise and thanks for your help
